internal/core/keys.py

```diff
@@ -7,8 +7,27 @@
 DAEMON_HOME = env.DAEMON_HOME
 
 
+def keys_add(account: str, multisig: bool = False, home: str = f"{DAEMON_HOME}-1"):
+    """The function `keys_add` will add a new key to the keyring.
+
+    Args:
+        account (_string_): wallet name or bech32 address.
+        home (_string_, optional): home directory of the node (default "{DAEMON_HOME}-1").
+
+    Returns:
+        _type_: _description_
+    """
+    command = f"{DAEMON} keys add {account} --home {home} --keyring-backend test --output json"
+    if multisig:
+        return exec_command(
+            command,
+            extra_args="--multisig=account1,account2,account3 --multisig-threshold 2",
+        )
```

Comment:
We shouldn't hardcode the multisig keys here. Consider taking it from user as an extra_args directly. Something like
```
def keys_add(account: str, extra_args: str = ""):

```